src/Flicking.ts

```diff
@@ -427,10 +427,14 @@ class Flicking extends Component {
    * flick.destroy();
    * console.log(flick.moveTo); // null
    */
-  public destroy(): void {
+  public destroy(option: Partial<DestroyOption> = {}): void {
```

Comment:
@WoodNeck 
Don't you think it's okay if you have one option and it's not an object?

Reply:
It's for flexibility.
Whenever we need to add more option for the `destroy()`, we just have to put more option in it.
This is because we might need more options for it, like should preserve only elements but not styles? or only styles? or so.